src/features/settings/screens/UsersScreen/components/UsersModal/UsersModal.tsx

```diff
@@ -45,11 +37,10 @@ const UsersModal = memo(({ onClose, onSubmitted }: UsersModalProps) => {
     // TODO: Create / Update users
     // eslint-disable-next-line no-console
     console.log(values);
-    setIsSubmitting(true);
     onSubmitted();
   };
 
-  return (
+  return !showModal ? null : (
```

Comment:
as it is necessary to wrap it in `!showModal` then we should update the FormModal so you cannot pass in the `visible` right? Because right now we will never pass in false to the visible prop

Reply:
Yes that makes sense, `visible` is no longer needed when we are not rendering the modal. I will fix it.